src/pymodaq_plugins_keithley/daq_viewer_plugins/plugins_0D/daq_0Dviewer_Keithley2100.py

```diff
@@ -152,22 +112,20 @@ def grab_data(self, Naverage=1, **kwargs):
         =============== ======== ===============================================
 
         """
+        logger.info("grab_data called")
```

Comment:
maybe add it as debug (not info) because it will be printed a lot on your log file which as the INFO level by default